ruby_event_store/spec/client_spec.rb

```diff
@@ -214,5 +215,17 @@ module RubyEventStore
       expect(subscriber.handled_events).to be_empty
     end
 
+    specify 'can handle protobuf event class instead of RubyEventStore::Event' do
+      client = RubyEventStore::Client.new(repository: InMemoryRepository.new)
+      event = ResTesting::OrderCreated.new(
+        event_id: "f90b8848-e478-47fe-9b4a-9f2a1d53622b",
```

Comment:
Is that ok that `event_id` here is a part of `data`?  It probably shouldn't be serialized, WDYT?

Reply:
@vbyno why not?

I think assuming this message may be serialized using protobuf and sent to a different system, it needs to contain the event id imho.

It was the assumption of the whole solution that the event message must have an id.

In protobuf message you don't distinguish between id, data, metadata. Developer can define the structure anyway they want.